accounts/admin.py

```diff
@@ -13,11 +13,12 @@ class CustomUserAdmin(UserAdmin):
     list_display = ('username', 'email', 'is_staff', 'is_active',)
     fieldsets = (
         (None, {'fields': ('username', 'password')}),
-        ('Personal info', {'fields': ('email', 'first_name', 'last_name', 'phone_number')}),
```

Comment:
Hope you have the phone number field back on the admin page. Haven't seen it.

Reply:
I have done that now